lib/formatter/binary-formatter.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+module.exports = (genericFormatter) => {
+    return genericFormatter.formatWithFallback((req, res, body, done) => {
+        if (!Buffer.isBuffer(body)) {
+            body = new Buffer(body.toString(), "binary");
```

Comment:
Wondering if we should have a buffer library or something of that nature, since every time we do a buffer it's binary, or a config option saying to have it in binary so we don't always have to write it and if we decide to use some other form of buffer format we can change it easier.